db/schema.rb

```diff
@@ -161,11 +161,11 @@
     t.boolean "disabled"
     t.boolean "employed", default: false
     t.string "employed_employer_name"
+    t.string "employed_employer_names", default: [], array: true
```

Comment:
Do you think these should eventually be pulled out into a separate table?

Reply:
yes perhaps! now that we have employer data that is different for SNAP and medicaid this is all getting pretty confusing.